DuckDuckGo/ContentBlocker.storyboard

```diff
@@ -163,31 +163,6 @@
                                             </constraints>
                                         </tableViewCellContentView>
                                     </tableViewCell>
-                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" selectionStyle="none" indentationWidth="10" id="QAm-5P-Wmg">
```

Comment:
Removing this cell leaves this popup a little large and goofy looking.  Setting "Use Preferred Explicit Size" with a height of 132 removes the whitespace and hides the bottom the cell's divider line nice.